desktop/libs/notebook/src/notebook/connectors/hiveserver2.py

```diff
@@ -227,7 +227,7 @@ def execute(self, notebook, snippet):
     try:
       if statement.get('statement_id') == 0:
         db.use(query.database)
-      handle = db.client.query(query)
+      handle = db.client.query(query, withMultipleSession=True)
```

Comment:
Some hive queries (for instance the ones dealing with the metastore) are run from method execute_and_wait in apps/beeswax/src/beeswax/server/dbms
Those don't need the extra session logic because they don't interfere with Tez jobs. The extra flag is only passed in user queries started from the notebook.